util/libcrypto.num

```diff
@@ -4857,3 +4857,21 @@ X509_LOOKUP_store                       4973	3_0_0	EXIST::FUNCTION:
 X509_STORE_load_file                    4974	3_0_0	EXIST::FUNCTION:
 X509_STORE_load_path                    4975	3_0_0	EXIST::FUNCTION:
 X509_STORE_load_store                   4976	3_0_0	EXIST::FUNCTION:
+EVP_RAND_CTX_new                        4977	3_0_0	EXIST::FUNCTION:
+EVP_RAND_CTX_free                       4978	3_0_0	EXIST::FUNCTION:
+EVP_RAND_name                           4979	3_0_0	EXIST::FUNCTION:
+EVP_RAND_provider                       4980	3_0_0	EXIST::FUNCTION:
+EVP_RAND_CTX_rand                       4981	3_0_0	EXIST::FUNCTION:
+EVP_RAND_CTX_reseed                     4982	3_0_0	EXIST::FUNCTION:
+EVP_RAND_CTX_strength                   4983	3_0_0	EXIST::FUNCTION:
+EVP_RAND_CTX_get_bytes                  4984	3_0_0	EXIST::FUNCTION:
+EVP_RAND_get_params                     4985	3_0_0	EXIST::FUNCTION:
+EVP_RAND_CTX_get_params                 4986	3_0_0	EXIST::FUNCTION:
+EVP_RAND_CTX_set_params                 4987	3_0_0	EXIST::FUNCTION:
+EVP_RAND_fetch                          4988	3_0_0	EXIST::FUNCTION:
+EVP_RAND_up_ref                         4989	3_0_0	EXIST::FUNCTION:
+EVP_RAND_free                           4990	3_0_0	EXIST::FUNCTION:
+EVP_RAND_gettable_params                4991	3_0_0	EXIST::FUNCTION:
+EVP_RAND_gettable_ctx_params            4992	3_0_0	EXIST::FUNCTION:
+EVP_RAND_settable_ctx_params            4993	3_0_0	EXIST::FUNCTION:
+EVP_RAND_do_all_ex                      4994	3_0_0	EXIST::FUNCTION:
```

Comment:
Do another rebase and change those numbers to question marks

Reply:
It's planned.  I need to rethink how I'm doing some of this.  The three DRBGs share a lot of common code to do with maximum sizes and reseed.  None of this is appropriate to seed sources.